elasticdl/python/common/tensor_utils.py

```diff
@@ -97,15 +100,17 @@ def pb_to_indexed_slices(pb):
 
 def serialize_indexed_slices(slices, pb):
     serialize_ndarray(slices.values, pb.concat_tensors)
-    if (
-        isinstance(slices.indices, np.ndarray)
-        and len(slices.indices.shape) > 1
-    ):
-        raise ValueError(
-            "IndexedSlices pb only accepts indices with one dimension, got %d",
-            len(slices.indices.shape),
-        )
-    pb.ids.extend(slices.indices)
+    indices = slices.indices
+    if isinstance(indices, np.ndarray):
+        if len(indices.shape) > 1:
+            raise ValueError(
+                "IndexedSlices pb only accepts indices with one"
```

Comment:
Add a white space at the end of the line. Otherwise, the output string will be "indices with onedimension" not "indices with one dimension"

Reply:
Done